packages/elements/src/containers/API.tsx

```diff
@@ -58,11 +56,17 @@ const propsAreWithDocument = (props: APIProps): props is APIPropsWithDocument =>
 };
 
 const APIImpl: React.FC<APIProps> = props => {
-  const { layout, apiDescriptionUrl } = props;
+  const { layout, apiDescriptionUrl = '' } = props;
   const apiDescriptionDocument = propsAreWithDocument(props) ? props.apiDescriptionDocument : undefined;
 
   const documentShouldBeFetched = apiDescriptionUrl && !apiDescriptionDocument;
-  const { data: fetchedDocument, error } = useSwr(documentShouldBeFetched ? apiDescriptionUrl! : null, fetcher);
+  const { data: fetchedDocument, error } = useQuery(
+    [apiDescriptionUrl],
+    () => fetch(apiDescriptionUrl).then(res => res.text()),
+    {
+      enabled: !!documentShouldBeFetched,
```

Comment:
Tiny suggestion: It'd be nicer if `documentShouldBeFetched` was an actual boolean instead.